manifests/plugin/remote_execution/ssh_user/params.pp

```diff
@@ -1,7 +1,6 @@
 # Remote Execution SSH user default parameters
```

Comment:
this files triggers:

```
manifests/plugin/remote_execution/ssh_user/params.pp:16:top_scope_facts:WARNING:top scope fact instead of facts hash
```